test/integration/test_app/sidecar-frontend/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM public.ecr.aws/amazonlinux/amazonlinux:2 AS builder
+RUN yum update -y && \
+    yum install -y ca-certificates unzip tar gzip git && \
+    yum clean all && \
+    rm -rf /var/cache/yum
+
+RUN curl -LO https://golang.org/dl/go1.17.1.linux-amd64.tar.gz && \
+    tar -C /usr/local -xzvf go1.17.1.linux-amd64.tar.gz
+
+ENV PATH="${PATH}:/usr/local/go/bin"
+ENV GOPATH="${HOME}/go"
+ENV PATH="${PATH}:${GOPATH}/bin"
+
+ENV GOPROXY=direct
```

Comment:
I have observed removing `ENV GOPROXY=direct` speeds up the build process. But I think its a small app and would not make much of a difference.

Reply:
good to know-- ill play around with it, thanks